source/_integrations/yale_smart_alarm.markdown

```diff
@@ -46,3 +53,21 @@ automation:
       target:
         entity_id: scene.OnArmedAway
 ```
+## Lock
+
+Lock platform requires a code for unlocking but no code for locking.
+The integration can be configured to provide a default code that is used if no code is supplied and the number of digits required for code.
+
+Example automation:
```

Comment:
These are pretty generic automations, do we really need to document these on integration level?

Reply:
You're probably right. There was an automation example in it from before so I just made one for lock too but I'll remove both as they don't add any benefit to the page really.